microshift_running_apps/microshift-gitops.adoc

```diff
@@ -0,0 +1,37 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-gitops"]
+= Automating application management with the GitOps agent
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-gitops
+
+toc::[]
+
+GitOps with ArgoCD for {microshift-short} is a lightweight, optional add-on agent derived from the Red Hat OpenShift GitOps Operator. GitOps for {microshift-short} uses the command-line interface (CLI) of Argo CD as the declarative GitOps engine. You can can consistently configure and deploy Kubernetes-based infrastructure and applications across clusters and development lifecycles.
+
+[id="microshift-gitops-can-do_{context}"]
+== What you can do with the GitOps agent
+By using the GitOps with Argo CD agent with {microshift-short}, you can leverage the following principles and tasks:
+
+* Create and manage your clusters and application configuration files using the core principles of developing and maintaining software in a Git repository. Using GitOps with ArgoCD you can update the single repository and GitOps automates the deployment of new applications or updates to existing ones.
```

Comment:
```suggestion
* Create and manage your clusters and application configuration files using the core principles of developing and maintaining software in a Git repository. You can update the single repository and GitOps automates the deployment of new applications or updates to existing ones.
```

"Using GitOps with ArgoCD" looks redundant as you have mentioned that part in the lead sentence.